lib/controllers/trees.js

```diff
@@ -20,6 +20,18 @@ module.exports = Router()
             next(err);
         }
     })
+    .get('/:id', async (req, res, next) => {
+        try {
+            const getTreesById = await Tree.receive(
```

Comment:
It looks like this is the GET all method. I think this should be the `receiveWithId` method :)